frontend/src/components/common/SectionHeader.stories.tsx

```diff
@@ -14,45 +14,166 @@
  * limitations under the License.
  */
 
+import { Icon } from '@iconify/react'; 
+import { Button, Chip, IconButton, Typography } from '@mui/material'; /
```

Comment:
this is causing all the frontend tests to fail
```suggestion
import { Button, Chip, IconButton, Typography } from '@mui/material';
```